<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountDetails</class>
 <widget class="QWidget" name="AccountDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Account Details</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="accountInformationGB">
     <property name="minimumSize">
      <size>
       <width>850</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Account Information</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="modifiedDateLabel">
          <property name="text">
           <string>Date Modified:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="modifiedDate">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="modifiedByLabel">
          <property name="text">
           <string>by</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="modifiedBy">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>167</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_29">
          <property name="text">
           <string>Date Created:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="createdDate">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="createdByLabel">
          <property name="text">
           <string>by</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="createdBy">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QGroupBox" name="detailsBox">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>275</height>
           </size>
          </property>
          <property name="title">
           <string>Details</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Name:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLineEdit" name="name"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Website:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLineEdit" name="website"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_26">
             <property name="text">
              <string>Ticker Symbol:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QLineEdit" name="tyckerSymbol"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Member of:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Ownership:</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Industry:</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Type:</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_28">
             <property name="text">
              <string>Campaign:</string>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="label_25">
             <property name="text">
              <string>Assigned to:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QComboBox" name="parentName">
             <item>
              <property name="text">
               <string/>
              </property>
             </item>
            </widget>
           </item>
           <item row="6" column="2">
            <widget class="QComboBox" name="industry">
             <item>
              <property name="text">
               <string/>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Apparel</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Banking</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Biotechnology</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Chemicals</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Communications</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Construction</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Consulting</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Education</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Electronics</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Energy</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Engineering</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Entertainment</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Environmental</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Finance</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Government</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Healthcare</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Hospitality</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Insurance</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Machinery</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Manufacturing</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Media</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Not For Profit</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Recreation</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Retail</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Shipping </string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Technology</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Telecommunication</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Transportation</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Utilities</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Other</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="8" column="2">
            <widget class="QComboBox" name="campaignName">
             <item>
              <property name="text">
               <string/>
              </property>
             </item>
            </widget>
           </item>
           <item row="7" column="2">
            <widget class="QComboBox" name="accountType">
             <item>
              <property name="text">
               <string/>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Analyst</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Competitor</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Customer</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Integrator</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Investor</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Partner</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Press</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Prospect</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Reseller</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Other</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="5" column="2">
            <widget class="QLineEdit" name="ownership"/>
           </item>
           <item row="9" column="2">
            <widget class="QComboBox" name="assignedUserName">
             <item>
              <property name="text">
               <string/>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="otherDetailsBox">
          <property name="minimumSize">
           <size>
            <width>275</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>Other Details</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Phone Office:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="phoneOffice"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Phone Fax:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="phoneFax"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Other Phone:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="phoneAlternate"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>Employees:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="employees"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Rating:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="rating"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Sic Code:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLineEdit" name="sicCode"/>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Annual Revenue:</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLineEdit" name="annualRevenue"/>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Email addresses</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QLineEdit" name="email1"/>
           </item>
           <item row="8" column="1">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>179</width>
               <height>48</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QGroupBox" name="addressesBox">
          <property name="minimumSize">
           <size>
            <width>275</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>Addresses</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Billing Address:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="billingAddressStreet"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>City:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="billingAddressCity"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>State:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="billingAddressState"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>Postal Code:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="billingAddressPostalcode"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>Country:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="billingAddressCountry"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_24">
             <property name="text">
              <string>Shipping Address</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLineEdit" name="shippingAddressStreet"/>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>City:</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLineEdit" name="shippingAddressCity"/>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>State:</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QLineEdit" name="shippingAddressState"/>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>Postal Code:</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QLineEdit" name="shippingAddressPostalcode"/>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="label_23">
             <property name="text">
              <string>Country:</string>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <widget class="QLineEdit" name="shippingAddressCountry"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="descriptionBox">
        <property name="title">
         <string>Description:</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QTextEdit" name="description"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>409</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="saveButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>&amp;Save</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>409</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>name</tabstop>
  <tabstop>website</tabstop>
  <tabstop>tyckerSymbol</tabstop>
  <tabstop>parentName</tabstop>
  <tabstop>ownership</tabstop>
  <tabstop>industry</tabstop>
  <tabstop>accountType</tabstop>
  <tabstop>campaignName</tabstop>
  <tabstop>assignedUserName</tabstop>
  <tabstop>phoneOffice</tabstop>
  <tabstop>phoneFax</tabstop>
  <tabstop>phoneAlternate</tabstop>
  <tabstop>employees</tabstop>
  <tabstop>rating</tabstop>
  <tabstop>sicCode</tabstop>
  <tabstop>annualRevenue</tabstop>
  <tabstop>email1</tabstop>
  <tabstop>billingAddressStreet</tabstop>
  <tabstop>billingAddressCity</tabstop>
  <tabstop>billingAddressState</tabstop>
  <tabstop>billingAddressPostalcode</tabstop>
  <tabstop>billingAddressCountry</tabstop>
  <tabstop>shippingAddressStreet</tabstop>
  <tabstop>shippingAddressCity</tabstop>
  <tabstop>shippingAddressState</tabstop>
  <tabstop>shippingAddressPostalcode</tabstop>
  <tabstop>shippingAddressCountry</tabstop>
  <tabstop>description</tabstop>
  <tabstop>saveButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
